From 5ff43dc3ed9b90589091f62b2e86e8454b47cac2 Mon Sep 17 00:00:00 2001
From: Adrien Kohlbecker <adrien.kohlbecker@gmail.com>
Date: Thu, 25 Oct 2018 23:08:18 +0100
Subject: [PATCH] Apply patches to source

---
 omnibus/config/software/datadog-agent.rb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/omnibus/config/software/datadog-agent.rb b/omnibus/config/software/datadog-agent.rb
index c259d935..b284736e 100644
--- a/omnibus/config/software/datadog-agent.rb
+++ b/omnibus/config/software/datadog-agent.rb
@@ -30,6 +30,8 @@ build do
   # include embedded path (mostly for `pkg-config` binary)
   env = with_embedded_path(env)
 
+  command "git am /root/0001-Fix-build-failure-on-32bit-armv7.patch"
+
   # we assume the go deps are already installed before running omnibus
   command "invoke agent.build --rebuild --use-embedded-libs --no-development", env: env
   if windows?
-- 
2.17.0

